src/Controller/UserController.php

```diff
@@ -57,9 +57,8 @@ public function new(Request $request): Response
     public function edit(Request $request): Response
     {
         // FIXME: Use user data from session or equivalent
-        $identificationNumber = '123456';
-        $user = new User();
-        $user->setIdentificationNumber($identificationNumber);
+        /** @var User $user */
+        $user = $this->getDoctrine()->getRepository(User::class)->findOneBy(['identificationNumber' => '9999999V']);
```

Comment:
If user is not found, $user->getIdentificationNumber() will throw error.
Can we test if $user is null ? If, yes, redirect to other previous page ?